aixplain/v2/agent.py

```diff
@@ -120,19 +118,3 @@ def create_custom_python_code_tool(cls, code: Union[str, Callable], description:
         from aixplain.factories import AgentFactory
 
         return AgentFactory.create_custom_python_code_tool(code=code, description=description)
-
```

Comment:
Sorry for the misunderstanding. Please add this piece of code back. This is actually a factory in version 2 of the SDK.